config/initializers/active_admin.rb

```diff
@@ -1,293 +0,0 @@
-ActiveAdmin.setup do |config|
```

Comment:
We should only remove instances where ActiveAdmin has been used to for authentication or the management of users. Removing ActiveAdmin completely from the codebase is something we should review in a wider context.

Reply:
I'm hoping we can get to a clean slate - leaving things like this lying around when they can't be used will create bloat.